src/SelectPanel/SelectPanel.tsx

```diff
@@ -26,7 +32,8 @@ interface SelectPanelBaseProps {
     open: boolean,
     gesture: 'anchor-click' | 'anchor-key-press' | 'click-outside' | 'escape' | 'selection'
   ) => void
-  placeholder?: string
+  title: string
+  inputLabel: string
```

Comment:
Let's make these optional for now to avoid a breaking release. cc @siddharthkp 
```suggestion
  // TODO: Make these required in the next major release
  title?: string
  inputLabel?: string
```

Reply:
nice! ship it

Reply:
Added!